serverless/packages/functions/src/trainspace/delete_trainspace.ts

```diff
@@ -0,0 +1,49 @@
+import { APIGatewayProxyHandlerV2 } from "aws-lambda";
+
+import { DynamoDBClient, DeleteItemCommand } from '@aws-sdk/client-dynamodb';
+import { DynamoDBDocumentClient } from '@aws-sdk/lib-dynamodb';
+
+export const handler: APIGatewayProxyHandlerV2 = async (event) => {
+    let queryParams = null;
+    if (event && (queryParams = event['pathParameters']) != null) {
+        const trainspace_id: string | undefined = queryParams['id'];
+
+        if (trainspace_id == undefined) {
+            return {
+                statusCode: 400,
+                body: JSON.stringify({ message : "Malformed request content - trainspace ID missing." }),
+            };
+        }
+        
+        const client = new DynamoDBClient({});
+        const documentClient = DynamoDBDocumentClient.from(client);
+        
+        const command = new DeleteItemCommand({
+            TableName : "trainspace",
+            Key :
+            {
+                "trainspace_id" :
+                {

```

Comment:
document client should automatically marshall input according to the docs: [https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-lib-dynamodb/](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-lib-dynamodb/)